Editor/App/AppWindow.cs

```diff
@@ -109,15 +115,97 @@ public NoCloseRenderForm(string title) : base(title)
             }
         }
 
-        private static void RebuildBackBuffer(ImGuiDx11RenderForm form, Device device, ref RenderTargetView rtv, ref Texture2D buffer, ref SwapChain swapChain)
+        private static void RebuildBackBuffer(Form form, Device device, ref RenderTargetView rtv, ref Texture2D buffer, ref SwapChain swapChain)
         {
             rtv.Dispose();
             buffer.Dispose();
             swapChain.ResizeBuffers(3, form.ClientSize.Width, form.ClientSize.Height, Format.Unknown, 0);
-            buffer = Texture2D.FromSwapChain<Texture2D>(swapChain, 0);
+            buffer = Resource.FromSwapChain<Texture2D>(swapChain, 0);
             rtv = new RenderTargetView(device, buffer);
         }
 
         private bool _isResizingRightNow;
+

```

Comment:
Just noticed I forgot to organize this stuff - but i guess now's my chance to properly ask you how you prefer variables and properties being organized in a class :)